.github/workflows/Docker.yaml

```diff
@@ -0,0 +1,31 @@
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+name: Docker
+
+on:
+  workflow_dispatch:
```

Comment:
add the `workflow_call:` trigger here (https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow)

Then in the `PostRelease` workflow you can just write:

```yaml
jobs:
  # ...
  docker:
    uses: ./.github/workflows/DemoContainer.yaml
```

So you don't have to duplicate the workflow.